code/obj/item/device/pda2/base_os.dm

```diff
@@ -560,6 +560,21 @@
 						if (href_list["norefresh"])
 							src.master.add_fingerprint(usr)
 							return
+					if("ack")
+						if(src.message_last + 20 > world.time) //Message sending delay
+							return
+
+						// group gets response message even if caller's PDA is destroyed or prankin'
+						var/message = "ACK: Responding"
+						src.pda_message(href_list["alert_group"], href_list["alert_group"], message, TRUE)
+						src.master.add_fingerprint(usr)
+
+						var/caller_id = href_list["alert_caller"]
+						if(!(caller_id in src.detected_pdas))
+							return
+						var/caller_name = detected_pdas[caller_id]
+						src.pda_message(caller_id, caller_name, message, FALSE)
```

Comment:
maybe only send a message directly to the caller if they're not in the alert group to cut down on chat spam?

Reply:
b9d2da1